Client/Frontend/Browser/Playlist/PlaylistPopoverView.swift

```diff
@@ -76,39 +76,38 @@ struct PlaylistPopoverView: View {
     
     private var addedToPlaylistView: some View {
         VStack(alignment: .leading) {
-            if #available(iOS 14.0, *) {
-                Text(Strings.PlayList.playlistPopoverAddedTitle)
-                    .foregroundColor(Color(UIColor.primaryButtonTint))
-                    .font(.title2.weight(.medium))
-                    .padding(.bottom, 20.0)
-            } else {
-                Text(Strings.PlayList.playlistPopoverAddedTitle)
-                    .foregroundColor(Color(UIColor.primaryButtonTint))
-                    .font(.headline.weight(.medium))
-                    .padding(.bottom, 20.0)
-            }
-            
-            Button(action: {
-                onPrimaryButtonPressed?()
-            }) {
+            VStack(spacing: 20.0) {
                 if #available(iOS 14.0, *) {
-                    Text(Strings.PlayList.playlistPopoverOpenInBravePlaylist)
-                        .frame(maxWidth: .infinity)
-                        .font(.title3.weight(.medium))
-                        .foregroundColor(.white)
-                        .padding()
+                    Text(Strings.PlayList.playlistPopoverAddedTitle)
+                        .foregroundColor(Color(UIColor.primaryButtonTint))
+                        .font(.title2.weight(.medium))
                 } else {
-                    Text(Strings.PlayList.playlistPopoverOpenInBravePlaylist)
-                        .frame(maxWidth: .infinity)
-                        .font(.body.weight(.medium))
-                        .foregroundColor(.white)
-                        .padding()
+                    Text(Strings.PlayList.playlistPopoverAddedTitle)
+                        .foregroundColor(Color(UIColor.primaryButtonTint))
+                        .font(.headline.weight(.medium))
+                }
+                
+                Button(action: {
+                    onPrimaryButtonPressed?()
+                }) {
+                    if #available(iOS 14.0, *) {
+                        Text(Strings.PlayList.playlistPopoverOpenInBravePlaylist)
+                            .frame(maxWidth: .infinity)
+                            .font(.title3.weight(.medium))
+                            .foregroundColor(.white)
+                            .padding()
+                    } else {
+                        Text(Strings.PlayList.playlistPopoverOpenInBravePlaylist)
+                            .frame(maxWidth: .infinity)
+                            .font(.body.weight(.medium))
+                            .foregroundColor(.white)
+                            .padding()
+                    }
```

Comment:
Don't know if there was something design specific here, but `BraveFilledButtonStyle` already handles font sizes + padding. Perhaps you wanted `.normal` instead of `.small`, either way not a big deal just an fyi that these shouldn't really be set